DualityPlugins/Tilemaps/Tilemaps/Components/ICmpTilemapRenderer.cs

```diff
@@ -47,6 +47,13 @@ public interface ICmpTilemapRenderer : ICmpRenderer
 		/// <returns></returns>
 		Point2 GetTileAtLocalPos(Vector2 localPos, TilePickMode pickMode);
 		/// <summary>
+		/// Gets the local position of the specified tile. The function does not check if the point is a valid tile position.
+		/// </summary>
+		/// <param name="tile">The position of the tile.</param>
+		/// <param name="alignment">Determines the point of the tile that is returned.</param>
+		/// <returns></returns>
+		Vector2 GetLocalPosAtTile(Point2 tile, Alignment alignment = Alignment.Center);
```

Comment:
I think the Alignment argument is a nice idea, but it shouldn't be part of the interface specification, as this will force all implementations to copy-paste the same logic. For now, I'd suggest to simply remove it because it's quite simple to calculate yourself if you have the tile size.

Instead, the position that should be returned is the upper left one.